### ISSUE TITLE
Update 2.22 has broken Next Up

### ISSUE THREAD

**Issue by lukehansford** [nixgates/plugin.video.seren#104]
Since the updated yesterday 2.22 next up is blank and does not load episodes.

Force syncing Trakt under tools does not fix it or rebuilding the database.

Even a reinstall of the add-on does not fix it.

**Comment by DJMix-I-Can** [comment]
I updated it on two devices. The first device was on a friends Firestick 4K device with Kodi 18.2. Next Up wasn't loading anything, so I rolled it back to 2.15. I couldn't check the rest of the folders because she had to leave. 
I'm currently on my Win 7 laptop with Kodi 17.6 because I was going to update a build. The time and date is correct. This is what I get with this installation. Finish watching does show TV Shows that I stopped watching and have removed from Trakt site. Collection does not retrieve anything. Watch List did show correct. Next up doesn't retrieve anything. Unfinished shows in collection doesn't even open up the next page as Next Up & Collection, it cancels out. Recent Episodes does show the correct some Next Up episodes for me BUT it's also showing episodes that I have already recently watched. Shows under My Shows List is showing correctly. I uninstalled 17.6 via Revo Uninstaller which also removes registry settings & installed 18.2 64bit. The time and date is still correct. Finish Watching doesn't show the same results as 17.6, it shows 90 percent less. Collection did retrieve my TV Shows collection. Watch List is correct. Next Up only showed three episodes but should have had eight episodes. Unfinished Shows is partially correct, missing a couple of shows. Recent Episodes is missing a lot of episodes especially since Cobra Kai's second season just came out, missing about 16 episodes. My Shows list is partially correct but still missing about 90 percent of the shows. I have a third device which is an Android MeCool M8S Pro L with Seren 0.1.35 and Kodi 18.2. I've learned to not update Seren as soon as it's released because there's always complications, no offense. As of right now, 0.1.35 is the best version for me and not going to update because it's currently fine for me. I activated debugging as I was checking each folder and uploaded the logs here https://drive.google.com/open?id=0B2b94BhFaqFgUGpjR3N3R2ZTTWM Hope it helps.
